From e1b023a6fc2818f871e3033bd21f72a2de7b7a21 Mon Sep 17 00:00:00 2001
Message-Id: <e1b023a6fc2818f871e3033bd21f72a2de7b7a21.1574207673.git.liu.denton@gmail.com>
In-Reply-To: <cover.1574207673.git.liu.denton@gmail.com>
References: <cover.1574125554.git.liu.denton@gmail.com>
	<cover.1574207673.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Tue, 19 Nov 2019 10:26:26 -0800
Subject: [PATCH v2 2/8] rev-list-options.txt: remove reference to --show-notes
To: Git Mailing List <git@vger.kernel.org>
Cc: Johannes Schindelin <Johannes.Schindelin@gmx.de>,
    Thomas Gummerer <t.gummerer@gmail.com>,
    Junio C Hamano <gitster@pobox.com>

In ab18b2c0df ("log/pretty-options: Document --[no-]notes and deprecate
old notes options", 2011-03-30), the `--show-notes` option was
deprecated. However, this reference to it still remains. Change it to
reference the replacement option: `--notes`.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 Documentation/rev-list-options.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/rev-list-options.txt b/Documentation/rev-list-options.txt
index 90ff9e2bea..311bc06a9b 100644
--- a/Documentation/rev-list-options.txt
+++ b/Documentation/rev-list-options.txt
@@ -58,7 +58,7 @@ endif::git-rev-list[]
 	`--all-match`).
 ifndef::git-rev-list[]
 +
-When `--show-notes` is in effect, the message from the notes is
+When `--notes` is in effect, the message from the notes is
 matched as if it were part of the log message.
 endif::git-rev-list[]
 
-- 
2.24.0.420.g9ac4901264

